gcs-action/src/main.ts

```diff
@@ -0,0 +1,61 @@
+import * as core from '@actions/core'
+import * as gcs from './gcs'
+async function run(): Promise<void> {
+  try {
+    const folderToUpload = core.getInput('folder-to-upload')
+    const fileToUpload = core.getInput('file-to-upload')
+    const bucketName = core.getInput('bucket-name')
```

Comment:
If any of these are required, mark them as so:


```ts
const bucketName = core.getInput('bucket-name', { required: true })
```

Reply:
Done